email.ppd

```diff
@@ -25,19 +25,19 @@
 
 *OpenUI *PageSize/Media Size: PickOne
 *OrderDependency: 0 AnySetup *PageSize
-*DefaultPageSize: Letter
+*DefaultPageSize: A4
 *PageSize A4/A4: "<</PageSize[595 842]/ImagingBBox null>>setpagedevice"
 *PageSize Letter/US Letter: "<</PageSize[612 792]/ImagingBBox null>>setpagedevice"
 *PageSize Legal/US Legal: "<</PageSize[612 1008]/ImagingBBox null>>setpagedevice"
 *CloseUI: *PageSize
 
-*DefaultPageRegion: Letter
+*DefaultPageRegion: A4
 *PageRegion A4/A4:		"<</PageSize[595 842]/ImagingBBox null>>setpagedevice"
 *PageRegion Letter/US Letter:	"<</PageSize[612 792]/ImagingBBox null>>setpagedevice"
 *PageRegion Legal/US Legal:	"<</PageSize[612 1008]/ImagingBBox null>>setpagedevice"
 
-*DefaultImageableArea: Letter
-*DefaultPaperDimension: Letter
+*DefaultImageableArea: A4
+*DefaultPaperDimension: A4
```

Comment:
This (and the ones above) should remain Letter as default. That's the kind of setting that should be adjusted after deployment.